lib/text/bidi_text.flow

```diff
@@ -26,19 +26,80 @@ export {
 
 	// Returns a reversed list with elements of original list wrapped into a pair with text flow direction
 	markListToBiDiReverse(list: List<?>, getText: (?) -> string) -> List<Pair<?, int>>;
+	CharCatSplitPosText(
+		text : string,
+		textAsInt : [int],
+		t_spaceAsInt : [int],
+		space2_tAsInt : [int],
+		space_tAsInt : [int]
+	);
+
+	makeCharCatSplitPosText(t : string) -> CharCatSplitPosText;
+}
+
+makeCharCatSplitPosText(t : string) -> CharCatSplitPosText {
+	t_space = " " + t;
+	space2_t = "  " + t;
+	space_t = t + " ";
+
+	lenText = strlen(t);
+	text = generate(0, lenText, \i -> getCharCodeAt(t, i));
```

Comment:
There is s2a or similar, which might be the same?

Reply:
done